analyzer-models/src/main/java/ffb/analyzer/models/espn/TransactionCounter.java

```diff
@@ -0,0 +1,66 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+
+public class TransactionCounter extends EspnEntity<TransactionCounter> {
+    private float acquisitionBudgetSpent;
+
+    @JsonProperty("acquisitions")
+    private int waiverAcquisitions;
+
+    @JsonProperty("drops")
+    private int droppedPlayerCount;
+
+    @JsonProperty("matchupAcquisitionTotals")
+    private Map<String, Integer> acquisitionsByScoringPeriod;
```

Comment:
It looks like these are stored by `weekId`, or something. Should this be a `Map<Integer, Integer>`?

Reply:
I kept it as String since all JSON keys are string, but I now realize that's unnecessary as Jackson will handle converting an int key to a string. So we should keep it as an Integer. Updated.